includes/class-wc-gateway-ppec-client.php

```diff
@@ -480,7 +480,7 @@ protected function _get_paypal_line_items_from_cart() {
 
 		$items = array();
 		foreach ( WC()->cart->cart_contents as $cart_item_key => $values ) {
-			$amount = round( $values['line_total'] / $values['quantity'] , $decimals );
+			$amount = round( $values['line_subtotal'] / $values['quantity'] , $decimals );
```

Comment:
why changing this to `line_subtotal` ? Same with the extra check added below.

The PR should reflect what it does, which is fixing the deprecated `WC_Cart::get_cart_url` otherwise it'd hard to understand the context from git history.

Reply:
My apologies @gedex, not sure how that snuck it's way into this PR... I have another one open (#345) and it appears to be the commit from there. I shall ammend this shortly to just include the cart fix.